### PR TITLE
Bumping versions due to a flaw found in serde_cbor-0.9.0

### PR DESCRIPTION
**Pull request by jensim**
Flaw found in transitive dependency failed my build
https://travis-ci.org/github/jensim/bitbucket_server_cli/builds/667567270
I added a audit step to the build as well, if you think its smart?

### CODE REVIEW

**Comment by jensim** [comment]
I love the initiative of having this dead-simple file-store  @seladb 

Maybe we could toggle nightly builds on travis? This would catch audit fails early on. 
Or add some mechanism for bumping versions with automagic

File: .travis.yml [hunk 1]
```diff
@@ -16,11 +16,13 @@ before_script:
   - rustup component add rustfmt
   - if [[ ${TRAVIS_RUST_VERSION} == stable ]]; then
       rustup component add clippy;
+      cargo install --force cargo-audit;
```
Comment (seladb) [inline .travis.yml]: I'm not sure we should enable cargo audit before we create a nightly build that will detect and update vulnerable packages. Do you know a tool that can:

- Run Travis on a nightly basis
- Detect vulnerable packages
- Bump the versions for these packages (this tool should know in which version the vulnerability was fixed) 
- Build and test pickledb to make sure nothing breaks
- Commit the changes to GitHub
- Release a new version

Until we do that, maybe we can just upgrade the versions as you did and release this minor version

Comment (jensim) [reply .travis.yml]: I couldn't find a crate to run dependency-version-bumping, after having looked for several minutes.
I dropped the changes to the build pipe, pending some automagic tool for dependency bumping

**Comment by seladb** [comment]
I've merged your PR and created a new release. Thank you so much for your contribution!

If you want to be included in the contributors list of this project please add the git email you committed your code with to your GitHub user. Currently your commit looks like this:

image

**Pull request merged**